sdcm/prometheus.py

```diff
@@ -181,7 +185,30 @@ def _publish_end_of_alerts(self, alerts: dict):
             if not alert.get('endsAt', None):
                 alert['endsAt'] = time.strftime("%Y-%m-%dT%H:%M:%S.0Z", time.gmtime())
             alert = updated_dict.get(alert['fingerprint'], alert)
-            PrometheusAlertManagerEvent.end(raw_alert=alert).publish()
+            labels = alert.get("labels") or {}
+            alert_name = labels.get("alertname", "")
+            node = labels.get("instance", "N/A")
+
+            event_filter \
+                .filter_by_node(node=node) \
+                .filter_by_starts_at(starts_at=alert.get("startsAt")) \
+                .filter_by_alert(alert=alert_name) \
+                .filter_by_period(period_type=EventPeriod.BEGIN.value)
+            begun_events = event_filter.get_filtered()
+            if not begun_events:
```

Comment:
Yes. Because of not all Prometheus events may be continuous AFAIR. I don't remember exactly example, but I think I saw such event. I am not sure 100% so I don't want to raise exception in this case